CHANGELOG.md

```diff
@@ -19,10 +19,17 @@ Compatibility:
 * Update `rb_str_modify` and `rb_str_modify_expand` to raise a `FrozenError` when given a frozen string (#2392).
 * Implement `rb_fiber_*` functions (#2402).
 * Implement `rb_str_vcatf`.
+* Add support for tracing allocations from C functions (#2403, @chrisseaton).
 
 Performance:
 
 * Moved most of `MonitorMixin` to primitives to deal with interrupts more efficiently (#2375).
+* Improved the performance of `rb_enc_from_index` by adding cached lookups. (#2379, @nirvdrum).
+* Improved the performance of many `MatchData` operations (#2384, @nirvdrum).
+* Significantly improved performance of TRegex calls by allow Truffle splitting (#2389, @nirvdrum).
+* Improved `String#gsub` performance by adding a fast path for the `string_byte_index` primitive (#2380, @nirvdrum).
+* Improved `String#index` performanc by addaing a fast path for the `string_character_index` primitive (#2383, @LillianZ).
```

Comment:
```suggestion
* Improved `String#index` performance by adding a fast path for the `string_character_index` primitive (#2383, @LillianZ).
```

Reply:
Thanks for the correction. I've amended the original commit with your fix to avoid this spilling out into multiple commits.